###################################################################
# Move the null terminator one char forwards.
# dave@slackbuilds.org
###################################################################
diff -Naur -x .git bb/src/protocol/gopher/gopher.c cc/src/protocol/gopher/gopher.c
--- bb/src/protocol/gopher/gopher.c	2020-01-14 03:24:11.259323801 +0000
+++ cc/src/protocol/gopher/gopher.c	2020-01-14 03:24:49.452454019 +0000
@@ -615,7 +615,7 @@
 	assert(line < end);
 
 	/* Just to be safe NUL terminate the line */
-	end[-2] = 0;
+	end[-1] = 0;
 
 	return line[0] == '.' && !line[1] ? NULL : line;
 }
